specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -30,6 +30,20 @@
                 "revisionName": "testcontainerApp0-ab1234",
                 "label": "production"
               }
+            ],
+            "ipSecurityRestrictionRules": [
+              {
+                "name": "name",
```

Comment:
Better name and desc in example :)

Reply:
updated

Reply:
This needs to be included in the response examples too.